great_expectations/core/yaml_handler.py

```diff
@@ -17,18 +21,17 @@ class YAMLHandler:
 
     Typical usage example:
 
-        simple_yaml: str = f\"""
+        simple_yaml: str = f'''
             name: test
             class_name: test_class
             module_name: test.test_class
-        \"""
+        '''
         yaml_handler = YAMLHandler()
         res: dict = yaml_handler.load(simple_yaml)
 
         example_dict: dict = dict(abc=1)
         yaml_handler = YAMLHandler()
         yaml_handler.dump(example_dict)
-
     """
```

Comment:
I think if you turn this into a code snippet it will render better, right now it's coming out like: 
![image](https://user-images.githubusercontent.com/9903066/213801320-980b71e3-eba4-4df8-9b26-e3f4e0c70658.png)

Reply:
The load and dump can also render better, take a look at the lists in `get_context()`

https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/util.py#L1796

Reply:
Done